### PR TITLE
Mobile-menu Peer Review

### PR DESCRIPTION
**Pull request by udberg**
The learning objectives of this project include 
- `The use of basic Javascript syntax.`
- `The use of Javascript to manipulate DOM elements.`
- `The use of Javascript events.`

I implemented the following features for the mobile version of this project

- `When the user clicks (or taps) the hamburger button, the mobile menu appears.`
- `When the user clicks (or taps) the close (X) button, the mobile menu disappears.`
- `When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.`
- `When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.`

The project design can be found [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,71 @@
...
+| [Getting Started](#getting-started-🛠) |
+| [Contributing](#contributing🛠) |
+| [Authors](#authors)         |
+| [License](#license)         |
+
+
+ ## Screenshot
+
+![img](./mockup)
+
```
Comment (DanSam5K) [inline README.md:25]: - [ ] kindly fix the screenshot image it seems to be broken.

File: css/header.css [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+}
+
+/* .overlay-header .closebtn {
+  position: absolute;
+  top: 20px;
+  right: 45px;
+  font-size: 60px;
+}
+
+/* Start query */
```
Comment (DanSam5K) [inline css/header.css]: - [ ] Would be nice you take out all commented code, as is often considered a good practice. 👍

**Review by DanSam5K — CHANGES_REQUESTED** [review]
## Hi @udberg, 🤝 

### Good job so far! 👏  🌱 
You have really put in a great amount of time and effort to make this project a reality.

** Highlights ** 💪 
- You really 🔨 nailed your pull request message and seems to have correctly made all linters check pass 👍 . Also, your commit history and project setup are great as well.
- But there are some issues that you still need to work on to go to the next project but you are almost there! ☺️ 

## Required Changes ♻️
_Check the comments under the review._
- [ ] Noticed when the user clicks (or taps) the close (X) button, the mobile menu seems not to disappears. kindly fix that 👍 
- [ ] When the user clicks (or taps) any of the mobile menu options, a correct part of the page is not displayed. You can do this: kindly check that also 🤗 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] [Optional] how about you adding more links menu to the project as implemented by the actual [Figma design](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/images/m7_mobile_menu/menu-mobile.png) for this project, what do you think? 🤔 

I would really ❤️  to see your next version of this project it would be amazing! 🤗 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BrianSammit — CHANGES_REQUESTED** [review]
-----

**Review by uwadonat — CHANGES_REQUESTED** [review]
Hi **@Udberg**, 

Great work on making the changes requested by a previous reviewer 👏  now mobile menu disappear when the user clicks (or taps) the close (X) button :mechanical_arm: :beer: 

You've done well implementing some of the requested changes, It shows that you are able. but there are still some which aren't addressed yet.

 ## Required Changes ♻️
- Kindly check the remaining changes requested by the previous reviewer and work on them. 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- Since the previous reviewer didn't notice, this requirement is optional but recommended to be done: 

- [ ] Please make sure that when the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.

   _Hope to see you celebrating soon_ :dancing_men: 
   Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**